Source/Core/Common/PcapFile.h

```diff
@@ -22,9 +22,14 @@
 class PCAP final
 {
 public:
+  static constexpr u32 PCAP_DATA_LINK_TYPE = 147;  // Reserved for internal use.
```

Comment:
You may want to place this into an enum class, along with the magic number used [here](#diff-2394256c451ee31017bb735ff6f6f762R94) to represent ethernet.